src/screens/PostScreen.tsx

```diff
@@ -45,19 +76,102 @@ export default class PostScreen extends React.Component<any, PostState>{
       getCommetsFromPostId(this.state.postId).then((data) => {
         //@ts-ignore
         this.setState({comments : data});
+        //@ts-ignore
+        this.setState({post_hanesst_id: data[0].hanesst_id})
+        console.log(data[0].hanesst_id)
       }); 
+    
     }
+   
+  }
+
+
+
+  handleSubmit = async ( e: React.FormEvent<HTMLFormElement>):Promise<void> => {
+      e.preventDefault();
+
+      if(this.state.username|| this.state.pwdHash || this.state.post_text || this.state.postId || this.state.post_hanesst_id){
+         //@ts-ignore
+      postNewComment(this.state.username, this.state.pwdHash, this.state.post_text, this.state.post_hanesst_id, this.state.comment_hanesst_id ).then(res => {
+        console.log(res);
+        });
+        
+        alert("Your comment was successfully posted! Please refresh the page to see it!");
+      } else {
+        alert("Something went wrong! Did you fill out all the inputs?");
+      }
+     
   }
 
+  newComment(){

```

Comment:
Again, you can render inside render() method, unless you are generating/mapping items to create JSX or html